frappe/custom/doctype/doctype_layout/doctype_layout.json

```diff
@@ -1,7 +1,7 @@
 {
  "actions": [],
  "allow_rename": 0,
- "autoname": "Prompt",
+ "autoname": "field:document_type",
```

Comment:
Why? then there is no point in creating Doctype Layout, Anyway, it will not allow to set auto name as document_type since document_type is not unique value